### PR TITLE
style(frontend): use ContentWithToolbar in DappModal

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

Uniform layout for Modals.

# Changes

- Use ContentWithToolbar in DappModals

# Tests

before:
image

image

after:

image

image

### CODE REVIEW

File: src/frontend/src/lib/components/dapps/DappModal.svelte [hunk 1]
```diff
@@ -46,110 +47,103 @@
...
-				})}</ExternalLink
-		>
-	{/if}
+		<svelte:fragment slot="toolbar">
+			{#if nonNullish(websiteURL)}
+				<ExternalLink
+					ariaLabel={replacePlaceholders($i18n.dapps.alt.open_dapp, {
+						$dAppName: dAppName
+					})}
+					styleClass="as-button primary padding-sm flex-1"
```
[the remark this answers is not in the document]
Comment (loki344) [reply src/frontend/src/lib/components/dapps/DappModal.svelte]: No, this is not on purpose, thanks for spotting it. updated screenshot

[the remark this answers is not in the document]
Comment (AntonioVentilii) [reply src/frontend/src/lib/components/dapps/DappModal.svelte]: Follow-up question, sorry if i didn't notice before: you are removing `flex`, will `flex-1` and the other `flex-` classes work? I always thought that for them to work you first need to define the container as `flex`

[the remark this answers is not in the document]
Comment (loki344) [reply src/frontend/src/lib/components/dapps/DappModal.svelte]: In fact, flex-1 (flex-shrink, flex-grow etc.) are for child elements of a display: flex container. see https://codepen.io/KittyGiraudel/pen/krMwzK/9a9ad8fb040f5efaf4e749b49cae7281 for a demo

[the remark this answers is not in the document]
Comment (loki344) [reply src/frontend/src/lib/components/dapps/DappModal.svelte]: the flex-row-reverse will still work as the externallink itself is inline-flex

[the remark this answers is not in the document]
Comment (loki344) [reply src/frontend/src/lib/components/dapps/DappModal.svelte]: btw I updated the screenshots, so we can see it works ;)

[the remark this answers is not in the document]
Comment (AntonioVentilii) [reply src/frontend/src/lib/components/dapps/DappModal.svelte]: ah perfect, thank you for double checking and the explanation!

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks

**Pull request merged**